static/js/components/subject_page/block.tsx

```diff
@@ -636,6 +637,15 @@ function renderTiles(
             displayValue={tile.answerMessageTileSpec.displayValue}
           />
         );
+      case "ANSWER_TABLE":
+        return (
+          <AnswerTableTile
+            key={id}
+            title={tile.title}
+            entities={tile.entities}
+            columns={tile.answerTableTileSpec.columns}
+          />
```

Comment:
supernit: alphabetize the props, should fix the codacy warning too

Reply:
done